os-server/src/main/java/app/openschool/common/security/JwtTokenProvider.java

```diff
@@ -0,0 +1,82 @@
+package app.openschool.common.security;
+
+import app.openschool.usermanagement.entities.UserPrincipal;
+import com.auth0.jwt.JWT;
+import com.auth0.jwt.JWTVerifier;
+import com.auth0.jwt.algorithms.Algorithm;
+import com.auth0.jwt.exceptions.JWTVerificationException;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.stereotype.Component;
+
+/** Useful Javadoc. */
+@Component
+public class JwtTokenProvider {
+
+  private static final long EXPIRATION_TIME = 432_000_000; // 5 days
+  private static final String AUTHORITIES = "authorities";
+  private static final String TOKEN_CANNOT_BE_VERIFIED = "Token can not be verified";
+
+  private String secret =
+      "ksfhlksjlkesjl;jksl;kfldskdl;ksfl;ks65456s465f4ds68f4dsf46ds4f5snkldj+dlksfnklslsjdkfljsdk";
+
+  /** Useful Javadoc. */
```

Comment:
please remove all redundant comments

Reply:
These Javadoc comments are required from Google's validation style.
Should I find that part in our checkstyle.xml file and remove to pass  ./gradlew checkstyle tasks.

Reply:
I think yes, Sargis jan, better nothing then something unuseful